src/lazy/binary/raw/v1_1/binary_buffer.rs

```diff
@@ -258,6 +258,17 @@ impl<'a> BinaryBuffer<'a> {
         Ok((value, remaining_input))
     }
 
+    pub fn read_fixed_uint_as_lazy_value(self, size_in_bytes: usize) -> ParseResult<'a, LazyRawBinaryValue_1_1<'a>> {
+        if self.len() < size_in_bytes {
+            return IonResult::incomplete("a uint8", self.offset());
+        }
```

Comment:
Is this function for reading `uint`s of a various sizes (hence the `size_in_bytes`) or for reading `uint8`s (as in the error message)?

Reply:
It is intended to support various sizes, but right now uint8 is the only thing implemented. Later PRs address this.